platform_secrets/service.py

```diff
@@ -142,73 +221,18 @@ async def get_all_secrets(
             ]
         return result
 
-    async def copy_to_namespace(
-        self,
-        org_name: str,
-        project_name: str,
-        target_namespace: str,
-        secret_names: list[str],
-    ) -> None:
-        """
-        Unwraps secrets from a dict and extracts them to a dedicated namespace.
-        """
-        secrets = await self.get_all_secrets(
-            with_values=True,
-            org_name=org_name,
-            project_name=project_name,
-        )
-        secrets_scope = set(secret_names)
-
-        missing_secret_names = secrets_scope - {secret.key for secret in secrets}
-        if missing_secret_names:
-            raise CopyScopeMissingError.create(missing_secret_names)
-
+    async def get_or_create_namespace(
+        self, org_name: str, project_name: str
+    ) -> dict[str, Any]:
```

Comment:
Let's not return dictionary but rather some dataclass. I think it should be added to apolo kube client